### PR TITLE
add .appveyor.yml

### PR DESCRIPTION
**Pull request by calebboyd**
re: https://github.com/sindresorhus/globby/issues/70#issuecomment-365316778

### CODE REVIEW

**Comment by kevva** [comment]
Maybe install a newer version of `npm` to make it faster on older Node.js versions? And also setting `fast_finish: true` since AppVeyor takes forever to run.

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+environment:
+  fast_finish: true
```
Comment (kevva) [inline .appveyor.yml]: I think this should be under `matrix` (not this one under environment, just add it to the bottom), e.g:

```yml
matrix:
  fast_finish: true
```

File: .appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  matrix:
+    - nodejs_version: "8"
+    - nodejs_version: "6"
+    - nodejs_version: "4"
+
+install:
+  - ps: Install-Product node $env:nodejs_version
+  - npm install npm@latest -g
```
Comment (kevva) [inline .appveyor.yml]: Nitpick, but better to be verbose with flags in code so it's easier to understand:

```
- npm install --global npm@latest
```

File: .appveyor.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+environment:
+  fast_finish: true
+  matrix:
+    - nodejs_version: "8"
```
Comment (kevva) [inline .appveyor.yml]: Single quotes.

Comment (cah-calebboyd) [reply .appveyor.yml]: I think there is an `.editorconfig` setting for that

**Comment by kevva** [comment]
Looks good 👍

File: .appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+environment:
+  matrix:
+    - nodejs_version: '8'
+    - nodejs_version: '8'
```
Comment (evocateur) [inline .appveyor.yml]: This line looks like a duplicate? Unless the intention was to test Node 8.x twice.

Comment (calebboyd) [reply .appveyor.yml]: 😬 🤦‍♂️

**Pull request closed without merging**